packages/message-backend/src/pdf/subscription-contract/translations.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
I reckon separating them to their respective locations makes the code base more cohesive. I fail to see the benefit of keeping them under one location in our instance.

Reply:
// import i18n from './../../i18n/init'

Because initialization can be made with one import and has translation fields set as needed. You can add new key to translation file,

This to fi.json and translations to sv.json and en.json

`"pdf" : { 
      "c4_2_3_daily": "vuorokauden",
      "c4_2_3_weekly": "viikon",
      "c4_2_3_monthly": "kuukauden",
      "c4_2_3_yearly": "vuoden",
      "abbr_daily": "vrk",
      "abbr_weekly": "vk",
      "abbr_monthly": "kk",
      "abbr_yearly": "v"
    }
}`

Reply:
Or you can make new initPdf.ts to has imports 
`import * as fi from '../translations/pdf/fi.json'
import * as en from '../translations/pdf/en.json'
import * as sv from '../translations/pdf/sv.json'`

and create new translation files

Reply:
I still fail to see the benefits here. What does it do beside complicate the code base? Doing translations for one email/pdf requires jumping between various folder locations and the translation keys are rather cryptic without the context of the email/pdf. The current overlap between the translations is also 0. If in the future, there is more overlap, then I do agree with keeping the translations in one place. Lastly, I'm pretty sure the initialization from one place has a potential issue with concurrency depending on how it is used: if for some reason someone decides to await something in middle of translating and then elsewhere the language is changed in-between. Of course, that is currently moreso a hypothetical situation.

Reply:
Moved the translations